br/use-br-command-line-tool.md

```diff
@@ -438,6 +438,39 @@ In the above command, `--table` specifies the name of the table to be restored.
 
 Restoring incremental data is similar to [restoring full data using BR](#restore-all-the-backup-data). Note that when restoring incremental data, make sure that all the data backed up before `last backup ts` has been restored to the target cluster.
 
+### Restore tables created in `mysql` database (experimental feature)
+
+BR backups tables created in `mysql` database by default.
+
+When you are restoring data using BR, the tables created in `mysql` database are not restored by default. If you need to restore them, explicitly contain them using [table filter](/table-filter.md#syntax). The following example restores `mysql.usertable` created in `mysql` database. The command restores `mysql.usertable` and other data at the same time.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url
+```
+
+In the above command, `-f '*.*'` is used to cover the default rules and `-f '!mysql.*'` instructs BR not to restore tables in `mysql` unless otherwise stated. `-f 'mysql.usertable'` indicates that `mysql.usertable` is required to be restored. For details, refer to [table filter document](/table-filter.md#syntax).
```

Comment:
```suggestion
In the above command, `-f '*.*'` is used to override the default rules and `-f '!mysql.*'` instructs BR not to restore tables in `mysql` unless otherwise stated. `-f 'mysql.usertable'` indicates that `mysql.usertable` is required for restore. For detailed implementation, refer to the [table filter document](/table-filter.md#syntax).
```